WORKSPACE

```diff
@@ -277,3 +277,9 @@ npm_install(
     name = "examples_hellohttp_npm",
     package_json = "//examples/hellohttp/nodejs:package.json",
 )
+
+# Register the default kubectl toolchain that expects the 'kubectl'
+# executable to be in the PATH
+register_toolchains(
```

Comment:
This should be moved to the k8s_repositories macro in k8s/k8s.bzl so that users of the k8s repo automatically get this toolchain registered for them. See https://github.com/bazelbuild/rules_docker/pull/559